manifests/00-cluster-role.yaml

```diff
@@ -101,8 +101,11 @@ rules:
   - featuregates
   - apiservers
   - networks
+  - clusterversions
   verbs:
   - get
+  - list
+  - watch
```

Comment:
I suppose this permission would be needed in the future for versioned featuregates, but why is it needed today?

Reply:
> I suppose this permission would be needed in the future for versioned featuregates, but why is it needed today?

Because the informer is started to be code compatible with the update comes.